identity-account/src/account/builder.rs

```diff
@@ -140,36 +136,52 @@ impl AccountBuilder {
     Ok(Arc::clone(self.storage.as_ref().unwrap()))
   }
 
-  /// Apply configuration to the IOTA Tangle client for the given [`Network`].
+  /// Sets the IOTA Tangle [`Client`], this determines the [`Network`] used by the identity.
+  ///
+  /// NOTE: this overwrites any [`ClientBuilder`] previously set by
+  ///       [`AccountBuilder::client_builder'].
```

Comment:
Intentional whitespace? :slightly_smiling_face:

Reply:
Yes but I suppose it doesn't matter.